2001-04-11  Jakub Jelinek  <jakub@redhat.com>

	* cpplex.c (_cpp_lex_token): Only warn if -Wcomment.

--- gcc/cpplex.c.jj	Fri Mar 30 11:44:42 2001
+++ gcc/cpplex.c	Tue Apr 10 18:33:34 2001
@@ -1017,7 +1018,7 @@ _cpp_lex_token (pfile, result)
 	    }
 
 	  /* Skip_line_comment updates buffer->read_ahead.  */
-	  if (skip_line_comment (pfile))
+	  if (skip_line_comment (pfile) && CPP_OPTION (pfile, warn_comments))
 	    cpp_warning_with_line (pfile, pfile->lexer_pos.line,
 				   pfile->lexer_pos.col,
 				   "multi-line comment");
